torch/_inductor/codegen/wrapper.py

```diff
@@ -468,8 +474,7 @@ def generate(self):
                 else:
                     self.wrapper_call.writeline(line)
 
-            output_refs = [x.codegen_reference() for x in V.graph.graph_outputs]
-            self.generate_return(output_refs)
+            self.generate_return()
```

Comment:
Would it be simpler to add a `get_output_refs()` to be overridden by subclasses? The result can be cached with `lru_cache` for good perf. Then, you don't have to change `self.generate_return`.

Reply:
Fixed as suggested.